cmd/snap-update-ns/system.go

```diff
@@ -66,9 +66,13 @@ func (upCtx *SystemProfileUpdateContext) Assumptions() *Assumptions {
 	//
 	// /snap/$SNAP_INSTANCE_NAME and /snap/$SNAP_NAME are added to allow
 	// remapping for parallel installs only when the snap has an instance key
+	//
+	// /var/lib/snapd/hostfs/var is there to allow the content interface to
+	// bridge to arbitrary sub-directories of /var from the host, even if they
+	// don't exist (they are created on the host in such case).
```

Comment:
the doc comment of this function needs changes too I think given this change inside